package.json

```diff
@@ -40,11 +43,22 @@
     "xo": "xo",
     "lint": "npm-run-all --parallel xo stylelint",
     "test": "npm run lint && npm run mocha",
-    "generate-test-results": "node-sass test/sass/ -o test/expected/",
-    "gulp-examples": "cd examples/less/gulp && gulp build && cd ../../postcss/gulp && gulp build && cd ../../scss/gulp && gulp build && cd ../../stylus/gulp && gulp build && cd ../../../..",
-    "node-examples": "node examples/less/node/index.js && node examples/postcss/node/index.js && node examples/scss/node/index.js && node examples/stylus/node/index.js"
+    "generate-test-results": "node-sass test/sass/ -o test/expected/ && prettier --write test/expected/*.css",
+    "examples": "npm-run-all gulp-examples node-examples",
+    "gulp-examples": "npm-run-all gulp-example-*",
+    "gulp-example-less": "gulp --gulpfile=examples/less/gulpfile.js",
+    "gulp-example-postcss": "gulp --gulpfile=examples/postcss/gulpfile.js",
+    "gulp-example-scss": "gulp --gulpfile=examples/scss/gulpfile.js",
+    "gulp-example-stylus": "gulp --gulpfile=examples/stylus/gulpfile.js",
+    "node-examples": "npm-run-all node-example-*",
```

Comment:
Could this and the gulp-examples be run in parallel maybe?

Reply:
I've parallelized the scripts and added `--continue-on-error` (These scripts are not meant to detect fails, since the input is the same as the test files).

Reply:
FYI `--continue-on-error` just means don't fail the task if one fails, but it will still fail at the end.